libs/wire-api/src/Wire/API/Routes/Public/Galley/Conversation.hs

```diff
@@ -693,6 +693,7 @@ type ConversationAPI =
                :> CanThrow 'GuestLinksDisabled
                :> CanThrow 'CreateConversationCodeConflict
                :> ZUser
+               :> ZHostOpt
```

Comment:
The versioning is unfortunately something to think about: The deployment is planned to happen mid of September. Will all clients use `v4` for this endpoint then? :thinking: 

For the `cargohold` part I've added the `ZHost` to all versions. Hand-wavy justifying that this is not really a change to the API because it happens between `nginz` and the backend.

Reply:
Right, I don't think we can rely on `v4` being used. I've updated also the v3 api. I think we can handwave this again with the same arguments.